datafusion/core/src/physical_plan/file_format/parquet/row_groups.rs

```diff
@@ -272,14 +281,14 @@ mod tests {
         let expr = logical2physical(&expr, &schema);
         let pruning_predicate =
             PruningPredicate::try_new(expr, Arc::new(schema)).unwrap();
-        let schema_descr = get_test_schema_descr(vec![(
-            "c1",
-            PhysicalType::INT32,
-            None,
-            None,
-            None,
-            None,
-        )]);
+        let schema_descr = get_test_schema_descr(vec![PrimitiveTypeField {
+            name: "c1",
```

Comment:
I am a big fan of explicit field names as a way to better document what is going on here.